code-collab-app/src/storage/peer_data.js

```diff
@@ -298,14 +315,14 @@ const peer_data = observable({
     },
 
     update_cell_lock(key, msg_type){
-        this.cell_lock_time = msg_type === LOCK_CELL ? new Date().getTime() : null;
+        this.random_value = msg_type === LOCK_CELL ? Math.floor(100000000 + Math.random() * 900000000) : null;
```

Comment:
Really unlikely, but what happens if the two random numbers are equal?

Reply:
The system will break ties by comparing their unique `peer_id` when two peers have the same random number. Added in the last commit.